tests/e2e/helpers/base.js

```diff
@@ -0,0 +1,179 @@
+import { ClientFunction, Selector } from 'testcafe';
+
+/**
+ * Runs all expected base tests for BookReader
+ *
+ * @param { BookReader } BR - Model
+ */
+const PAGE_FLIP_WAIT_TIME = 1000;
+export function runBaseTests (BR) {
+  test('On load, pages fit fully inside of the BookReader™', async t => {
+    await t.expect(BR.shell.visible).ok();
+    await t.expect(BR.BRcontainer.visible).ok();
+
+    const shellHeight = await BR.shell.getBoundingClientRectProperty('height');
+    const bookHeight = await BR.BRcontainer.getBoundingClientRectProperty('height');
+    await t.expect(bookHeight).lte(shellHeight, 'images do not get cropped vertically');
+
+    const shellWidth = await BR.shell.getBoundingClientRectProperty('width');
+    const bookWidth = await BR.BRcontainer.getBoundingClientRectProperty('width');
+    await t.expect(bookWidth).lte(shellWidth, 'images do not get cropped horizontally');
+  });
+
+  test('Nav menu displays properly', async t => {
+    const { Nav } = BR;
+
+    await t.expect(Nav.desktop.goPrevious.visible).ok();
+    await t.expect(Nav.desktop.goNext.visible).ok();
+    await t.expect(Nav.desktop.mode1Up.visible).ok();
+    await t.expect(Nav.desktop.mode2Up.visible).ok();
+    await t.expect(Nav.desktop.modeThumb.visible).ok();
+    await t.expect(Nav.desktop.zoomIn.visible).ok();
+    await t.expect(Nav.desktop.zoomOut.visible).ok();
+    await t.expect(Nav.desktop.fullScreen.visible).ok();
+  });
+
+  test('2up mode - Clicking `Previous page` changes the page', async t => {
+    const { Nav, BRcontainer} = BR;
+    const onLoadBR = BRcontainer.child(0);
+    const initialImages = onLoadBR.child('img');
+    const origImg1Src = await initialImages.nth(0).getAttribute('src');
+    const origImg2Src = await initialImages.nth(-1).getAttribute('src');
+
+    const previousButton = Nav.desktop.goPrevious;
+    await t.click(previousButton);
+    await t.wait(PAGE_FLIP_WAIT_TIME); // wait for animation and page flip to happen
+
+    const nextBrState = await Selector('.BRcontainer').child(0);
+    const prevImages = nextBrState.child('img');
```

Comment:
We have a new selector call because we need to see the changed state that happened in the DOM.  The initial `.BRcontainer` selector points only to the loaded state.